beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -167,14 +172,125 @@ def normalize(tensor):
 
 print("Median relative difference between original and MuLaw reconstucted signals: {:.2%}".format(err))
 
+######################################################################
+# ``torchaudio`` also supports various ``augmentations``:
+# 
+# -  **TimeStretching**: Wrapping and allowing for batching for the `phase_vocoder` functional.
```

Comment:
For the description, you can simply use the one provided in the docstring. In theory, we've design torchaudio so that all transforms should be wrappers of a corresponding functional.